docs/en/user_guides/visualization.md

```diff
@@ -1,8 +1,30 @@
 # Visualization
 
+- [Single Image](#single-image)
 - [Browse Dataset](#browse-dataset)
 - [Visualizer Hook](#visualizer-hook)
 
+## Single Image
+
+`demo/image_demo.py` helps the user to visualize the prediction result of a single image, including the skeleton and heatmaps.
+
+```shell
+python demo/image_demo.py ${IMG} ${CONFIG} ${CHECKPOINT} [-h] [--out-file OUT_FILE] [--device DEVICE] [--draw-heatmap]
+```
+
+| ARGS                  | Description                      |
+| --------------------- | -------------------------------- |
+| `IMG`                 | The path to the test image.      |
+| `CONFIG`              | The path to the config file.     |
+| `CHECKPOINT`          | The path to the checkpoint file. |
+| `--out-file OUT_FILE` | Path to output file.             |
+| `--device DEVICE`     | Device used for inference.       |
+| `--draw-heatmap`      | Visualize the predicted heatmap. |
+
+Here is an example of SimCC visualization:
+
+![simcc_visualization](https://user-images.githubusercontent.com/13503330/218374365-009f540e-98db-4d29-9bcc-943596d71ea9.png)
```

Comment:
I am a little concerned that SimCC visualization can be confusing to some beginners. Should we use a simple 2D heatmap example here, considering that this is a rather entry-level document?